src/stories/components/MainMenu/MainMenu.js

```diff
@@ -45,16 +45,16 @@ export const MainMenu = () => {
 
   return (
     <Router>
-      <ul className='main-menu'>
+      <ul className="main-menu">
         {
           icons.map((icon) => {
             return (
-              <li key={icon}>
+              <li className="main-menu__item" key={icon}>
                 <NavLink
                   to={icon}
                   className={'menu-item__link'}>
-                  <p className='menu-item__icon'>{getIcon(icon)}</p>
-                  <p className='menu-item__text'>{icon}</p>
```

Comment:
@venjason `<p>` are block elements, which are semantically incorrect when nested inside an `inline` element such as a link. You can use `span` for that!